src/app/query-selector/page.module.css

```diff
@@ -0,0 +1,19 @@
+.title {
+  font-size: 40px;
+  font-weight: 600;
+  font-family: 'Inter', sans-serif;
+  margin: 5px 20px;
+  color: #ffffff;
+  display: inline-block;
+  background-color: #216869;
+  padding: 4px 30px;
+  border-radius: 15px;
+}
+.header {
+  background-color: #eeeffb;
+}
+
+.mainPage {
+  padding-top: 20px;
+  background-color: #eeeffb;
```

Comment:
We start out on the first page (landing page) with clean/simple color theming and then transition to sort of a different feel on the second page (query-selector). I would generally recommend going with defaults (we have no need for theming this application in a specific way).
But! If you'd like to play around with colors, I would recommend selecting a cohesive existing color scheme across the whole application rather than specifying colors directly with hex values. Mantine uses open color as default: https://mantine.dev/theming/colors/ which gives you different shades of basic colors - you could choose a primary color to apply shades of across the application or maybe a primary and a secondary if you think you're going to want more variation.